docs/manifest.json

```diff
@@ -1068,72 +1073,72 @@
     },
     {
       "title": "Security",
-      "description": "Security advisories",
+      "description": "Learn about security advisories.",
       "path": "./security/index.md",
       "icon_path": "./images/icons/security.svg",
       "children": [
         {
           "title": "API tokens of deleted users not invalidated",
-          "description": "Fixed in v0.23.0 (Apr 25, 2023)",
+          "description": "Learn about the fix in v0.23.0 (Apr 25, 2023).",
           "path": "./security/0001_user_apikeys_invalidation.md"
         }
       ]
     },
     {
       "title": "FAQs",
-      "description": "Frequently asked questions",
+      "description": "Learn about frequently asked questions.",
       "path": "./faqs.md",
       "icon_path": "./images/icons/info.svg"
     },
     {
       "title": "Guides",
-      "description": "Employee-authored tutorials",
+      "description": "Learn from employee-authored tutorials.",
```

Comment:
It seems like most of these just add "Learn from" or "Learn about" which just elongates the descriptions?

Reply:
These are meta tags used for SEO. Would you rather "Learn from" be prefixed in the Google description? I certainly think it adds clarity.